tests/index.test.ts

```diff
@@ -67,6 +67,7 @@ it('should export configs that refer to actual rules', () => {
 		expect(rule.startsWith(ruleNamePrefix)).toBe(true);
 		expect(ruleNames).toContain(ruleName);
 
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
I tried to change this code so that we wouldn't need the eslint-disable comment at all, but oh welp, that didn't work 😅 So let's let this be.

Reply:
This is a test, so I'd say it's fine having an exception here.